src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,51 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private K[] keyArray;
+    private V[] valueArray;
+    private int storegeSize = 0;
```

Comment:
```suggestion
    private int size;
```
You do not need to add 'storage' to var names. We are in Storage class, so, all variables belongs to Storage